cmd/sriov-network-config-daemon/start.go

```diff
@@ -123,10 +127,27 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 
 	sriovnetworkv1.AddToScheme(scheme.Scheme)
 	mcfgv1.AddToScheme(scheme.Scheme)
+	configv1.Install(scheme.Scheme)
 
 	snclient := snclientset.NewForConfigOrDie(config)
 	kubeclient := kubernetes.NewForConfigOrDie(config)
 	mcclient := mcclientset.NewForConfigOrDie(config)
+	openshiftFlavor := utils.OpenshiftFlavorDefault
```

Comment:
thats used to construct Openshift context to my understanding. can we move this logic to a "NewOpenshfitContext(config)" or similar method part of openshift_context.go ?